src/ivc/protogalaxy/mod.rs

```diff
@@ -523,7 +523,7 @@ mod verify_chip {
             .map_err(|err| Error::BetasStroke { err })
     }
 
-    /// Evaluate the values of the Lagrange polynomial zero for a cyclic subgroup of length `n` (`2.pow(log_n)`) at
+    /// Evaluate the values of the Lagrange polynomial for a cyclic subgroup of length `n` (`2.pow(log_n)`) at
```

Comment:
Unlike prover, verifier don't need to evaluate one lagrange poly over multiple points. i.e. From what circuit should do is to evaluate:

(1) when evaluate poly K: L_0(gamma)
(2) when fold all instances together: L_i(gamma) for all i

but after I double check It seems this is what the code doing. But the comment here is quite mis-leading

Reply:
For not rebase all PR only for rustdoc, let's discuss here:
```rust
    /// Evaluate the Lagrange polynomial at `cha` point
    ///
    /// You can look at [`fft::get_omega_or_inv`] to see how the target cyclic group is formed
    ///
    /// # Mathematical Representation
    ///
    /// ```math
    /// L_i(X)=\frac{\omega^i}{n}\frac{X^n-1}{X-\omega^i}
    /// ```
    /// where {1, \omega, \omega^2, ..., \omega^n} - cyclic group, check [`iter_cyclic_subgroup`] for
    /// more details
    ///
    /// # Generics
    /// - `T` is setup for main gate
    /// - `L`: 'Length' - constant representing the number of instances to
    ///                   fold in a single `prove`.
    ///
    ///                   `L+1` be power of two.
    ///
    ///                   Lagrange domain will be builded with:
    ///                   [`crate::nifs::protogalaxy::poly::PolyContext::get_lagrange_domain`] with
    ///                   TRACES_LEN=L
```

Is that less confusing doc?